src/app/services/xml-parsers/bibliography-parser.service.ts

```diff
@@ -0,0 +1,96 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { forkJoin, Observable } from 'rxjs';
+import { map, shareReplay } from 'rxjs/operators';
+import { AppConfig } from 'src/app/app.config';
+
+function getStringFromTags(tagLabel: string, cit: Element) {
+  let res = '';
+  Array.from(cit.getElementsByTagName(tagLabel)).map(element => res += element.textContent.replace(/\s+/g, ' ') + ' ');
+
+  return res.trim();
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class BibliographyParserService {
+  private editionUrls = AppConfig.evtSettings.files.editionUrls || [];
+  private findedCits = { complete: 0, total: 0 };
+
+  constructor(
+    private http: HttpClient,
+  ) {
+  }
+
+  public getBibliographicCitations(): Observable<CitationsFeature> {
+    const parser = new DOMParser();
+    const bibliographicCitations: BibliographicCitation[] = [];
+
+    return forkJoin(this.editionUrls.map((path) => this.http.get(path, { responseType: 'text' }))).pipe(
+      map(responses => {
+        responses.map(response => {
+          Array.from(parser.parseFromString(response, 'text/xml').getElementsByTagName('bibl')).map(citation => {
```

Comment:
Are you sure that this code works for each case? What about complex bibliographic entries?
Moreover you have to consider only the bibliography, that is inside a `<listBibl>` element. Read the TEI guidelines better.

Reply:
_answering the first:_ Yes, it works. A complex bibliographic entry has `x` titles, `y` authors, and `k` dates inside, and I have handled these situations, also doing tests, and everything seems to work perfectly. 
_about the second matter:_ I thought about it initially. The problem is that I should take all the `<listBibl>` with `<bibl>` inside. As I did, I take all the `<bibl>` of the whole document skimming the doubles. 
Said in metalanguage:
with the change you suggest, I would do 2 `get`, one inside the other, while as I did I do only one.